### DISCUSSION TITLE
5.4 download link

### DISCUSSION THREAD

**Discussion by jennnital** [endink/Mediapipe4u-plugin#169] [General]
Hi, 

I'd like to know if the 5.4 download link is available! I'd love to try it out soon. Thanks! 看到readme 更新了 5.4 但是找不到下载链接。 谢谢！

**Comment by endink** [comment]
Thanks for your interesting, wait for the last issue #124 fixed, I will release it.

**Comment by endink** [comment]
BTW, new features and UE5.4 support were done, but I want to fix #124. 

It's a complex issue, and fixing it took longer time than I expected.

**Comment by jennnital** [comment]
Thank you!
